exercises/G-conditionals-2/exercise-4.js

```diff
@@ -8,7 +8,13 @@
   Hint: Google how to check if a string contains a word
 */
 
-function containsCode(sentence) {}
+function containsCode(sentence) {
+  if (sentence.includes('code')){
+    return true
+  } else {
+    return false
+  }
```

Comment:
Good job Joy. While this solution is correct, you can get the same result by usting 
return sentence.includes('code');
as the result will be true ot false without the need for if function